engine/item_effects.asm

```diff
@@ -2313,6 +2316,37 @@ ItemWasntUsedMessage:
 	jp PrintText
 ; f7e8
 
+Ball_ReplacePartyMonCaughtBall:
+	ld b, PARTYMENUACTION_00
+	call UseItem_SelectMon
+	jp c, ItemNotUsed_ExitMenu
+
+	ld a, [wCurItem]
+	ld b, a
+	ld a, [wCurPartyMon]
+	ld hl, wPartyMon1CaughtBall
+	call GetPartyLocation
+	ld a, [hl]
+	and CAUGHTBALL_MASK
+	cp b
+	
+	jp z, AlreadyInThatBallMessage
+	ld a, [hl]
+	and -CAUGHTBALL_MASK-1 ; One's complement
+	add b
+	ld [hl], a
+	call UseDisposableItem
+	ld hl, BallReplacedText
+	jp PrintText
+
+BallReplacedText:
+	text "Ball changed."
```

Comment:
It would be nice if this were more descriptive. Like "Transferred <mon name> to the <item name>!" Or at least "Stored in the <item name>!"